src/config.zig

```diff
@@ -231,6 +233,7 @@ pub const configs = struct {
             .cache_accounts_size_default = @sizeOf(vsr.tigerbeetle.Account) * 1024 * 1024,
             .cache_transfers_size_default = @sizeOf(vsr.tigerbeetle.Transfer) * 2048,
             .cache_transfers_posted_size_default = @sizeOf(u256) * 256 * 1024,
+            .cache_account_history_size_default = 2 * 256 * 1024,
```

Comment:
Is there a better way to do that?
The ideal would be to define ZERO for the minimal possible cache 🤔